mmseg/datasets/pipelines/loading.py

```diff
@@ -135,8 +135,9 @@ def __call__(self, results):
             backend=self.imdecode_backend).squeeze().astype(np.uint8)
         # modify if custom classes
         if results.get('label_map', None) is not None:
+            gt_semantic_seg_copy = gt_semantic_seg.copy()
```

Comment:
```suggestion
            # Add deep copy to solve bug of repeatedly
            # replace `gt_semantic_seg`, which is reported in 
            # https://github.com/open-mmlab/mmsegmentation/pull/1445/
            gt_semantic_seg_copy = gt_semantic_seg.copy()
```

Reply:
@Yzichen  Hi, thanks for your great pr. I suggest add comment to let users know why we add deep copy and give related pr info.